### DISCUSSION TITLE
Подарок в виде разбана на новый год

### DISCUSSION THREAD

**Discussion by Unknow123456789** [octothorp-team/public-dobrograd#821] [Предложения]
Привет! Я предлагаю всем людям у кого бан на год не за рдм, а за нарушение каких-либо правил можно будет разбанить или снизить срок в новый год.

**Comment by Hardcore-punk** [comment]
Поддерживаю! но все таки поправку на рдм можно сделать, все таки кто осознал свою ошибку имеет место быть тут
